orm/model/ormtable/table.go

```diff
@@ -156,4 +157,7 @@ type AutoIncrementTable interface {
 	// InsertReturningPKey inserts the provided entry in the store and returns the newly
 	// generated primary key for the message or an error.
 	InsertReturningPKey(ctx context.Context, message proto.Message) (newPK uint64, err error)
+
+	// LastInsertedSequence retrieves the last inserted sequence in the table.
```

Comment:
```suggestion
	// LastInsertedSequence retrieves the sequence number of the last entry inserted into the table.
```

Reply:
Also it will be zero if no entries have been inserted right? We should document that